doc/reference/network_physical.md

```diff
@@ -36,6 +36,7 @@ Key                             | Type      | Condition             | Default
 `bgp.peers.NAME.address`        | string    | BGP server            | -                         | Peer address (IPv4 or IPv6) for use by `ovn` downstream networks
 `bgp.peers.NAME.asn`            | integer   | BGP server            | -                         | Peer AS number for use by `ovn` downstream networks
 `bgp.peers.NAME.password`       | string    | BGP server            | - (no password)           | Peer session password (optional) for use by `ovn` downstream networks
+`bgp.peers.NAME.holdtime`       | string    | BGP server            | `180`                     | Peer session holdtime (optional)
```

Comment:
Same as above.

Reply:
Can you help me resolve spellcheck test. Should I simply add holdtimer word to .wordlist.txt file?

Reply:
No, it should be "hold time" (two words) in the text, as in the other occurrence. The parameter itself is fine since it's marked as code.

Reply:
Didn't wait for your reply for an answer and corrected, sorry.
But already returned the file to its previous correct state.
I will be more patient in the future.

Reply:
Thanks, this should work now.